source/ed/connectors/gtfs_parser.cpp

```diff
@@ -639,8 +639,6 @@ void CalendarGtfsHandler::finish(Data& data) {
 }
 
 void CalendarGtfsHandler::handle_line(Data& data, const csv_row& row, bool) {
-    if (row.empty())
```

Comment:
There is another bunch of `row.empty()` in the file that would need some cleanups.

Reply:
:+1: